library/src/main/scala/scala/scalajs/js/package.scala

```diff
@@ -105,4 +109,75 @@ package object js {
       "type of a Scala.js library has been called. This is most likely " +
       "because you tried to run Scala.js binaries on the JVM. Make sure you " +
       "are using the JVM version of the libraries.")
+
+  /** Allows to type safely cast an instance of a Scala class to facade trait
```

Comment:
Missing `.` at the end of the line.

"type safely cast" is a bit weird. I suggest "to cast an instance ... in a type-safe way."